schedule/migration/migration_offline_scc_deregistration_ppc64le.yaml

```diff
@@ -0,0 +1,41 @@
+name: migration_offline_scc_deregistration_ppc64le.yaml
+description: |
+  This is for offline migraiton tests with deregistration on ppc64le.
+vars:
+  ORIGIN_SYSTEM_VERSION: '%HDDVERSION%'
+  UPGRADE_TARGET_VERSION: '%VERSION%'
+schedule:
+  - migration/version_switch_origin_system
+  - boot/boot_to_desktop
+  - update/patch_sle
+  - console/check_system_info
+  - migration/record_disk_info
+  - migration/reboot_to_upgrade
+  - migration/version_switch_upgrade_target
+  - installation/isosize
+  - installation/bootloader
+  - installation/welcome
+  - installation/upgrade_select
+  - installation/scc_registration
+  - installation/addon_products_sle
+  - installation/resolve_dependency_issues
+  - installation/installation_overview
+  - installation/disable_grub_timeout
+  - installation/start_install
+  - installation/await_install
+  - installation/logs_from_installation_system
+  - installation/reboot_after_installation
+  - installation/grub_test
+  - installation/first_boot
+  - migration/post_upgrade
+  - console/system_prepare
+  - console/consoletest_setup
+  - console/zypper_lr
+  - console/check_os_release
+  - console/check_system_info
+  - console/hostname
+  - console/force_scheduled_tasks
+  - console/scc_deregistration
```

Comment:
I think we can add condition to control these modules loaded or not for publish qcow or not, so this yaml file can be used for two phase of continuous migration test.

Reply:
As we discussed. The second part uses online migration. They cannot share the same yaml file.

Reply:
Updated